src/components/dropdown/dropdown.e2e.ts

```diff
@@ -539,8 +540,9 @@ describe("calcite-dropdown", () => {
 
     it("control max items displayed", async () => {
       const maxItems = 7;
-      const page = await newE2EPage({
-        html: html`<calcite-dropdown max-items="${maxItems}">
+      const page = await newE2EPage();
+      await page.setContent(html`
+        <calcite-dropdown max-items="${maxItems}">
```

Comment:
This doesn't look related to the changes in this PR, but more a refactor of how the page content is set.  Can you revert this back to the way it was since the shorthand method is preferred?